src/engine/source/hlp/src/parsers/kvmap.cpp

```diff
@@ -67,76 +211,94 @@ Parser getKVParser(const Params& params)
 
     return [sep, delim, quote, esc, name = params.name, targetField](std::string_view txt)
     {
-        std::string_view kvInput = txt;
+        std::vector<std::string> tokens;
```

Comment:
The original implementation of this parser mixes the semantic and sintactic parser. should be a good idea split them